### ISSUE TITLE
Missing Middleman logo

### ISSUE THREAD

**Issue by lowmess** [CSS-Tricks/serverless#14]
It appears there is no `static/images/middleman.svg`, which leads to the alt text showing on that card. While it should be easy enough to drop the SVG into place, it may be a good idea to also [create a generic placeholder style for broken images](https://bitsofco.de/styling-broken-images/) on resource cards.

![Middleman card shows alt text](https://user-images.githubusercontent.com/6688324/58287995-44bf4500-7d67-11e9-846d-7d2c92fd1449.png)

~This screenshot also shows another issue (`props.title` not being properly interpolated in alt text).~

**Comment by ara303** [comment]
According to [Middleman's website](https://middlemanapp.com) the logo is one brow M and a second white M. On the Serverless site, the card backgrounds are white so it wouldn't work. 

[Here's the original SVG.](https://middlemanapp.com/images/middleman-logo.svg) The brown is actually black at 45% opacity, but I think the intended effect is that it be `#8a6d28`.

For [the site's favicon](https://middlemanapp.com/favicon.ico), this issue is neatly sidestepped by putting their yellow background colour behind the logo.

I'm not sure what is best to do. Either change the second M to the yellow used in the background, or edit the image and add the yellow background behind it like the favicon... (That's a bit beyond me at the moment.)

---

**Edit:** I've changed the SVG data to use the brown hex code as above for the first M and the site's background yellow for the second. Github won't let me attach the SVG file to this issue, so here's a raster and the code beneath:

<img width="422" alt="Screenshot 2019-05-24 at 12 03 03 pm" src="https://user-images.githubusercontent.com/769754/58323543-31ba7c80-7e1c-11e9-8dcc-cf82eba30632.png">

````svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 400 340" class="middleman-logo" aria-labelledby="middleman-logo__title" role="img">
  <title id="middleman-logo__title">Middleman</title>
  <path xmlns="http://www.w3.org/2000/svg" class="middleman-logo__top-left-bar" fill="#8a6d28" d="M0 40L200 0v30L0 60z"/>
  <path class="middleman-logo__top-right-bar" fill="#fbc547" d="M200 0l200 40v20L200 30z"/>
  <path xmlns="http://www.w3.org/2000/svg" class="middleman-logo__left-m" fill="#8a6d28" d="M0 78v184l45 5V152l45 83 47-83v129l53 7V52l-57 8-43 83-43-70z"/>
  <path class="middleman-logo__right-m" fill="#fbc547" d="M400 78v184l-45 5V152l-45 83-47-83v129l-53 7V52l57 8 43 83 43-70z"/>
  <path xmlns="http://www.w3.org/2000/svg" class="middleman-logo__bottom-left-bar" fill="#8a6d28" d="M0 300l200 40v-30L0 280z"/>
  <path class="middleman-logo__bottom-right-bar" fill="#fbc547" d="M200 340l200-40v-20l-200 30z"/>
</svg>
````

Any thoughts? :)

**Comment by chriscoyier** [comment]
I just pushed one up with the same idea as the favicon you pointed out. Seems the most on-brand.
